x-pack/plugins/cases/public/components/use_breadcrumbs/index.ts

```diff
@@ -20,8 +20,8 @@ const casesBreadcrumbTitle: Record<ICasesDeepLinkId, string> = {
   [CasesDeepLinkId.casesCreate]: i18n.translate('xpack.cases.breadcrumbs.create_case', {
     defaultMessage: 'Create',
   }),
-  [CasesDeepLinkId.casesConfigure]: i18n.translate('xpack.cases.breadcrumbs.configure_cases', {
-    defaultMessage: 'Configure',
+  [CasesDeepLinkId.casesConfigure]: i18n.translate('xpack.cases.breadcrumbs.settings', {
```

Comment:
I had made this change and later reverted because few navigation variables are still with `configure` name and it is confusing (like in `x-pack/plugins/cases/public/common/navigation/hooks.ts`)

changing all variables from `configure` to `settings` is a bigger change. 
Do you think we can handle this in other PR where we change all variables?

Reply:
Yes, ofc. I thought it was much simpler 🙂 . We can leave it as a tech dept.